lib/backend/k8s/resources/customresource.go

```diff
@@ -116,45 +115,58 @@ func (c *customK8sResourceClient) Update(kvp *model.KVPair) (*model.KVPair, erro
 	})
 	logContext.Debug("Update custom Kubernetes resource")
 
-	// Convert the KVPair to the K8s resource.
-	resIn, err := c.converter.FromKVPair(kvp)
-	if err != nil {
-		logContext.WithError(err).Info("Error updating resource")
-		return nil, err
-	}
+	// Create storage for the updated resource.
+	resOut := reflect.New(c.k8sResourceType).Interface().(CustomK8sResource)
 
-	fmt.Printf("\n\n**** in kvp: %v\n\n", kvp)
+	providedRV := ""
```

Comment:
We may or may not need to add a test where we send in an `Update` that has a `Revision` set on it.  Not sure if we already have one of those tests somewhere.